src/lib/annotations/Annotator.js

```diff
@@ -832,6 +831,28 @@ class Annotator extends EventEmitter {
         return this.currentAnnotationMode === mode;
     }
 
+    /**
+     * Scrolls specified annotation into view
+     *
+     * @private
+     * @param {Object} data - annotation data for thread that should scroll into view
+     * @param {string} data.threadID - annotation thread id
+     * @param {string} data.page - annotation page number
+     * @return {void}
+     */
+    scrollToAnnotation(data) {
+        const { threadID, page } = data;
+        if (!threadID || !page) {
```

Comment:
Should we be displaying or emitting some kind of error if incorrect params are passed into this method? Say if the threadID corresponds to a thread that is not on the specified page?

Reply:
I'd almost say that we should have some sort of invalidation occur if a thread doesn't exist on a page, which in turn will potentially delete the annotation. Thoughts? Is it possible for an annotation to be saved onto a page that it doesn't belong to?

Reply:
I guess the only time I could see that occurring is if annotations were created from the API. I don't think the thread would be deleted in this case, there would just be no action, or indication that anything even happened

Reply:
Since annotation page numbers are stored in the details blob in the DB, there's not really any validation that the API would do if "invalid" annotations were created. When rendering that annotation on our end, it would trigger a notification at the top saying "some annotations failed to load" so maybe triggering similar behavior if someone tries to scroll to something that doesn't exist on the file?

@varunmaker would it make sense to not display any kind of error here or show some kind of custom annotations error here?

Reply:
Removing the page number from the emitted event. This should prevent any kind of invalidation checks that would needed.